daemon/api.go

```diff
@@ -220,8 +221,9 @@ var (
 	}
 
 	usersCmd = &Command{
-		Path: "/v2/users",
-		GET:  getUsers,
+		Path:     "/v2/users",
+		GET:      getUsers,
+		RootOnly: true,
```

Comment:
Do we have a test for this? When I remove this line (RootOnly: true) it seems no test fails(?).

Reply:
I've added a test for this, at least to catch it if we remove it by mistake